dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops

```diff
@@ -175,6 +175,18 @@ elements={0}{1} elements
 
 enable_batch_operations_as_true_not_supported=WLS native CMP setting "enable-batch-operations-as-true" is not directly supported in EclipseLink CMP. See migration doc for details.
 
+encryptor_decrypt_old_algorithm=Database password used in {0} was encrypted by deprecated algorithm.
```

Comment:
Minor suggested changes:
- Add "The" before database ("The database password")
- Add "a" before deprecated ("by a deprecated algorithm")
- Change "re-encrypt" to "reencrypt"
- Remove "It is recommended to" ("Reencrypt it by")

After these changes, the message is:
"The database password used in {0} was encrypted by a deprecated algorithm.
Reencrypt it by `passwordUpdate.sh` from eclipselink.zip bundle."